Dev/config/strings.js

```diff
@@ -74,6 +77,7 @@ export default {
     EditAssignment: 'Edit Assignment',
     EnterAssignmentHere: 'Enter assignment here...',
     OutStanding: 'Outstanding!',
+    EnterAssignment: 'Enter Assignment',
```

Comment:
We should change all our messaging to sentence casing (only capitalize the first letter) instead of title casing (capitalize all letters) unless something is clearly a title, which most of the strings aren't. Right now we are inconsistent, we use one casing in some messages and another in other messages.